src/NextGen/DonationForm/FormTemplates/ClassicFormTemplate/js/template.ts

```diff
@@ -1,6 +0,0 @@
-window.givewp.template.mount({
```

Comment:
@jonwaldstein is this mount not necessary if since there are no custom field templates?

Reply:
Yup exactly! I commented on the developer template but referenced this so you can see.